<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255,255,255);</string>
        </property>
        <property name="text">
         <string>일 매출 현황</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="date_info">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>39</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
color: rgb(255,255,255);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>743</width>
           <height>399</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(200, 200, 200);</string>
            </property>
            <property name="rowCount">
             <number>61</number>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>168</number>
            </attribute>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <column>
             <property name="text">
              <string>주문 항목</string>
             </property>
             <property name="background">
              <color>
               <red>114</red>
               <green>141</green>
               <blue>199</blue>
              </color>
             </property>
            </column>
            <column>
             <property name="text">
              <string>가격</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>새 행</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>합계</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>427</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255,255,255);</string>
        </property>
        <property name="text">
         <string>총 매출</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="total_sum">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
color: rgb(255,255,255);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
